treenode.go

```diff
@@ -673,7 +673,7 @@ func (n *TreeNodeInstance) SendToChildrenInParallel(msg interface{}) error {
 // returns it with any error that might have happened during the creation. This
 // protocol is only handled by onet, no service are "attached" to it.
 func (n *TreeNodeInstance) CreateProtocol(name string, t *Tree) (ProtocolInstance, error) {
-	pi, err := n.overlay.CreateProtocol(name, t, NilServiceID)
+	pi, err := n.overlay.CreateProtocol(name, t, n.Token().ServiceID)
```

Comment:
Lol. That was easy ;)

Reply:
First I wanted to submit only this line - then I thought that I have 2 engineers that will tell me to write some test...